dockerfile-configs/common-components.yaml

```diff
@@ -131,7 +131,9 @@
       VERSION="v1.17.0";\
       curl -fsSLO https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-$VERSION-linux-amd64.tar.gz;\
       tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin;\
-      rm -f crictl-$VERSION-linux-amd64.tar.gz
+      rm -f crictl-$VERSION-linux-amd64.tar.gz;\
+      rm -rf /etc/crictl.yaml;\
```

Comment:
Maybe we should not delete an existing configuration on the node. What do you think about only adding it when it is not available yet?

Reply:
i checked this file status before cp the new one - the installation process of `crictl` doesn't create config if you don't run it, after you first time run `crictl` it will create an `/etc/crictl.yaml` which contains one line of `null` in it.
the default config will point to `unix:///var/run/dockershim.sock` which is not desired `containerd`

what do you think?

Reply:
What I mean is that, whenever an operator launches another ops/toolbelt pod, this will override the current /etc/crictl.yaml file on the host. 
But thinking about it, that might be even better.

Reply:
Follow up on this: there are cases when the node image already got crictl installed- so overriding an already existing config should be avoided. PN me for more details.